spark-derived-feature-view/apps/main.py

```diff
@@ -0,0 +1,126 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from datetime import timedelta
+
+from feathub.feathub_client import FeathubClient
+from feathub.feature_tables.sinks.file_system_sink import FileSystemSink
+from feathub.feature_views.feature import Feature
+from feathub.feature_views.derived_feature_view import DerivedFeatureView
+from feathub.feature_views.transforms.over_window_transform import (
+    OverWindowTransform,
+)
+
+from feathub.common import types
+from feathub.feature_tables.sources.file_system_source import FileSystemSource
+from feathub.table.schema import Schema
+
+
+if __name__ == "__main__":
+    client = FeathubClient(
+        props={
+            "processor": {
+                "type": "spark",
+                "spark": {"master": "spark://spark-master:7077", "native.spark.executor.memory": "500M"},
```

Comment:
Why do we need the `"native.spark.executor.memory": "500M"` configuration"? The e2e example should have been simple enough to use a negligible amount of memory.

Reply:
I moved this config into docker-compose.